pkg/reconciler/revision/table_test.go

```diff
@@ -83,7 +83,7 @@ func TestReconcile(t *testing.T) {
 		WantStatusUpdates: []clientgotesting.UpdateActionImpl{{
 			Object: rev("foo", "first-reconcile",
 				// The first reconciliation Populates the following status properties.
-				WithLogURL, AllUnknownConditions, MarkDeploying("Deploying")),
+				WithLogURL, AllUnknownConditions, MarkDeploying("Deploying"), MarkResourcesUnavailable("PodAutoscalerHasNotReconciledYet", "PodAutoscaler with class  has not been reconciled yet")),
```

Comment:
Status string seems weird here:
```PodAutoscaler with class  has not been reconciled yet```
does it make sense to check if the key exists. If it does not exist create show different message. 

I came across an issue where webhook was not registered and default autoscaler class was not set which can result into similar error message.

Reply:
Planning to add status like
`NotReconciled` 

Can you please eloberate `does it make sense to check if the key exists` ?  i mean about which key?

Reply:
I am taking about [line](https://github.com/knative/serving/pull/5560/files#diff-83d81ad460499e87cc0d935f33c5228eR146)

Check if the key `autoscaling.ClassAnnotationKey` exists.

Reply:
Because of [SetDefaults](https://github.com/knative/serving/blob/master/pkg/apis/autoscaling/v1alpha1/pa_defaults.go#L37) key `autoscaling.ClassAnnotationKey` and default value `autoscaling.KPA` always exist. so i feel no need to check key existence.

If user send empty value for annotation key `autoscaling.knative.dev/class` then it will display
```
PodAutoscaler with class  has not been reconciled yet
```
(there is a extra space in between `class` and `has` which shows user dint provide any value for the class)

Reply:
Print it in with a `%q` maybe to make it more obvious?